presto-tests/src/test/java/com/facebook/presto/tests/TestTpchLocalStats.java

```diff
@@ -237,4 +237,12 @@ public void testDateComparisons()
                 checks -> checks
                         .estimate(OUTPUT_ROW_COUNT, defaultTolerance()));
     }
+
+    @Test
+    public void testVarcharComparisons()
```

Comment:
Did you consider adding unit test for `LiteralInterpreter.toExpression` besides/instead?